index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+        rel="stylesheet">
+    <link rel="stylesheet" href="index.css">
+    <title>mobile skeleton</title>
+</head>
+<body>
+<main>
+    <div class="full-wrapper">
+        
+        <nav>
+            <div class="imgs">
+                <img src="/images/Light.png" alt="">
+            </div>
+            <div class="nav-wrapp">
+                <div class="logo">
+                    <span>My Logo</span>
+                </div>
+                <div class="right-nav">
+                    <i id="menu" class="material-icons">menu</i>
+                </div>
+            </div>
+        </nav>
+    
+        <div class="content-wrapper">
+            <div class="header">
+                <span>I’m Desmond
+                    Glad to see you!</span>
+            </div>
+            <div class="about">
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </div>
```

Comment:
- [ ] Please wrap this in a `p` tag 👍 . It is a description and has many words.  In such a case, the `p` is highly recommended 🏄